doc/man/8/ceph-monstore-tool.rst

```diff
@@ -0,0 +1,89 @@
+:orphan:
+
+======================================================
+ ceph-monstore-tool -- ceph monstore manipulation tool
+======================================================
+
+.. program:: ceph-monstore-tool
+
+Synopsis
+========
+
+| **ceph-monstore-tool** <store path> <cmd> [args|options]
+
+
+Description
+===========
+
+:program:`ceph-monstore-tool` is similar to `ceph-kvstore-tool`. It allows
+users to manipulate MonitorDBStore's data (monmap, osdmap, etc.) offline.
+The default RocksDB debug level is `0`. This can be changed using `--debug`.
+
+Please Note:
+    Ceph-specific options should be in the format `--option-name=VAL`
+    (specifically, do not forget the '='!!)
+    Command-specific options need to be passed after a `--`
+    e.g., `get monmap --debug -- --version 10 --out /tmp/foo`
+
+Commands
+========
+
+:program:`ceph-monstore-tool` utility uses many commands for debugging purpose
+which are as follows:
+
+:command:`store-copy <path>`
+    Copies the store to PATH.
+
+:command:`get monmap [-- options]`
+    Get monmap (version VER if specified) (default: last committed).
+
+:command:`get osdmap [-- options]`
+    Get osdmap (version VER if specified) (default: last committed).
+
+:command:`get msdmap [-- options]`
+    Get msdmap (version VER if specified) (default: last committed).
+
+:command:`get mgr [-- options]`
+    Get mgrmap (version VER if specified) (default: last committed).
+
+:command:`get crushmap [-- options]`
+    Get crushmap (version VER if specified) (default: last committed).
+
+:command:`get osd_snap <key> [-- options]`
+    Get osd_snap key (`purged_snap` or `purged_epoch`).
```

Comment:
> I've backported this to the Reef release branch. I'll ask whether it should also be backported to the Quincy release branch, and if it should be, then I will backport it there too.

@zdover23 We shouldn't be taking and especially backporting doc updates ahead of actual code changes.  `get osd_snap` and `rm` commands, as well as `--debug` option are still pending in https://github.com/ceph/ceph/pull/52855 and their appearance may change.  Whenever you see a comment like

> Note: The docs here include #52855 changes.

on a doc PR, I would suggest to hold the merge.

Reply:
@idryomov: Roger that, @idryomov. I'll start keeping a list of pending changes of this type, and I will do my best to make the consideration of that list a part of the ethos of Ceph upstream documentation.